<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CenterGLWidget" name="mainGLWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <zorder>refresh_shader_btn</zorder>
    <zorder>refresh_shader_btn</zorder>
   </widget>
   <widget class="QTabWidget" name="tabWidget_control">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>181</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_shading">
     <attribute name="title">
      <string>显示</string>
     </attribute>
     <widget class="QPushButton" name="refresh_shader_btn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh Shader</string>
      </property>
     </widget>
     <zorder>refresh_shader_btn</zorder>
     <zorder>mainGLWidget</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadOBJ"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadOBJ">
   <property name="text">
    <string>LoadOBJ</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CenterGLWidget</class>
   <extends>QWidget</extends>
   <header>centerglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
